src/screens/guides/components/brush-and-zoom/ecology.md

```diff
@@ -0,0 +1,177 @@
+# Brush and Zoom
+
+**IMPORTANT: VictoryZoomContainer replaces VictoryZoom**
```

Comment:
This seems a little scary/loud, especially if you've never used either VictoryZoomContainer or VictoryZoom (which will be the lion's share of readers pretty soon). Maybe just put a note in the next paragraph, like so:

> Use `VictoryZoomContainer` as your containerComponent to add panning and zooming behavior to any Victory components that work with an x, y coordinate system. (Note: VictoryZoom was replaced by VictoryZoomContainer.)